src/runner/browser-env/vite/browser-modules/constants.ts

```diff
@@ -18,3 +18,6 @@ export const SOCKET_MAX_TIMEOUT = 2147483647;
 export const SOCKET_TIMED_OUT_ERROR = "operation has timed out";
 
 export const MAX_ARGS_LENGTH = 50;
+
+export const ASYMMETRIC_MATCHER =
```

Comment:
I suggest to add a comment here: https://github.com/jestjs/jest/blob/726ca20752e38c18e20aa21740cec7aba7891946/packages/pretty-format/src/plugins/AsymmetricMatcher.ts#L11